lib/src/config.rs

```diff
@@ -203,7 +213,7 @@ impl fmt::Display for ConfigNamePathBuf {
 /// constrained by the source type.
 pub trait ToConfigNamePath: Sized {
     /// Path type to be converted from `Self`.
-    type Output: Borrow<ConfigNamePathBuf>;
+    type Output: Borrow<ConfigNamePathBuf> + Into<ConfigNamePathBuf>;
```

Comment:
Is `ToOwned` more appropriate here? (I don't know the answer.)

Reply:
No. It's micro optimization to move owned (or parsed) `ConfigNamePathBuf` without cloning. `.to_owned()` doesn't consume `self`, but `.into()` can.